app/server.go

```diff
@@ -2117,3 +2025,11 @@ func (a *App) GetAppliedSchemaMigrations() ([]model.AppliedMigration, *model.App
 func (s *Server) Platform() *platform.PlatformService {
 	return s.platform
 }
+
+func (s *Server) Log() *mlog.Logger {
```

Comment:
Now we have both `s.Log()` and `s.GetLogger()` maybe we should consolidate them into one (I'm fine with that being `s.Log()`. 2/5

Reply:
Good point 💯